src/bindings/python/requirements_test.txt

```diff
@@ -35,6 +35,7 @@ pytest-html==1.19.0
 py>=1.9.0
 radon
 retrying
+torch
```

Comment:
can we test extensions in frontend tests?
we want to keep python tests as framework independent as possible

Reply:
I can see that we have frontend tests for onnx, tf, paddle in this directory so I thought it'd be a good place. Also the tests I did serve as a python bindings tests for pytorch frontend.

Reply:
but we don't have dependency on TF, PADDLE.
If you managed to write tests without adding torch dependency, I'm totally OK to keep it here.

The idea to avoid more and more dependencies because of specific use cases.

Reply:
ok, I moved those tests of out src/bindings/python/tests.